tests/test_conditions.py

```diff
@@ -6,177 +6,200 @@
 from qiskit_qasm3_import import types, ConversionError
 from qiskit_qasm3_import.data import Symbol, Scope
 from qiskit_qasm3_import.expression import resolve_condition
+from qiskit_qasm3_import.converter import State
 
 
 def _equal(left: ast.Expression, right: ast.Expression):
     return ast.BinaryExpression(ast.BinaryOperator["=="], left, right)
 
 
+def _make_context(symbols=None):
+    context = State(Scope.GLOBAL)
```

Comment:
Pretty sure that this a bug; the only positional argument to `State` is the `source` string.

Reply:
Yes. this is another relic.

Reply:
Fixed in 68c6541